### PR TITLE
Move localizations to i18n

### PR DESCRIPTION
**Pull request by brianclio**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,13 +41,8 @@ enumerated_column :state, { open: 1, closed: 2, failed: 3 }, selections: { not_f
...
 
-# You can also add localization options for these select helper outputs. These are matched to the name of the selection.
-# These are optional and provide support for a default value.
-default_localizations = { failed: "OMG WTF BBQ!?" }
-all_states_localizations = { open: "I am open" }
-
-enumerated_column :state, { open: 1, closed: 2, failed: 3 }, selections: { all_states: [1, 2, 3] }, localizations: { default: default_localizations, all_states: all_states_localizations }
-TestClass::ALL_STATES # [["I am open", 1], ["Closed", 2], ["OMG WTF BBQ!?", 3]]
+# You can also add i18n localizations for these select helper outputs. 
+# The gem matches values in the helpers with i18n entries under .models.class_name.column_name.scope_name(or default).key
```
Comment (brianclio) [inline README.md]: there is probably a better way to put this info in the README

File: config/application.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
+module RailsColumnEnumerator
```
Comment (fimmtiu) [inline config/application.rb]: This file needs to be deleted.

File: lib/rails_column_enumerator.rb [hunk 3]
```diff
@@ -9,10 +9,10 @@ def self.get_int(val, enum_values, valid_values = nil)
     val
   end
 
-  def self.get_record(val, enum_values, valid_values, localization_values, helper_name)
+  def self.get_record(val, enum_values, valid_values, helper_name, column_name, class_name)
     val = get_int(val, enum_values, valid_values)
     key = valid_values[val]
-    text = localization_values[helper_name].try(:[], key) || localization_values[:default].try(:[], key) || valid_values[val].to_s.titleize
+    text = I18n.t(".models.#{class_name}.#{column_name}.#{helper_name}.#{key}", :default => I18n.t(".models.#{class_name}.#{column_name}.default.#{key}", :default => valid_values[val].to_s.titleize))
```
Comment (tylersuzukinelson) [inline lib/rails_column_enumerator.rb]:
```
text = I18n.t(".models.#{class_name}.#{column_name}.#{helper_name}.#{key}", default: I18n.t(".models.#{class_name}.#{column_name}.default.#{key}", default: valid_values[val].to_s.titleize))
```

File: lib/rails_column_enumerator/version.rb [hunk 4]
```diff
@@ -1,3 +1,3 @@
 module RailsColumnEnumerator
-  VERSION = "1.3.0"
+  VERSION = "1.3.1"
 end
```
Comment (tylersuzukinelson) [inline lib/rails_column_enumerator/version.rb]: Missing EOF newline character

File: spec/support/en.yml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+en:
+  models:
+    test_class:
+      sample_column:
+        default:
+          first: "Yay, first"
+          next: "Aww, second"
+        first_two:
+          first: "Number one"
```
Comment (tylersuzukinelson) [inline spec/support/en.yml]: Missing EOF newline character

Comment (brianclio) [inline spec/support/en.yml]: ```¯\_(ツ)_/¯```

File: lib/rails_column_enumerator/version.rb (hunk 4 above)
Comment (brianclio) [inline lib/rails_column_enumerator/version.rb]: and?

File: lib/rails_column_enumerator.rb (hunk 3 above)
Comment (brianclio) [inline lib/rails_column_enumerator.rb]: it's a lot clearer to just say "remove rocket syntax" or something

File: README.md (hunk 1 above)
Comment (tylersuzukinelson) [inline README.md]:
```
# I18n localizations can be added for these select helper outputs.
# The gem will match values in the helpers with I18n entries stored in:
# .models.class_name.column_name.scope_name.key
```

Comment (brianclio) [inline README.md]: doesn't really make the message any clearer, was thinking of a nice way to show a sample usage or something

Comment (tylersuzukinelson) [inline README.md]: Perhaps including the example from the specs?

File: lib/rails_column_enumerator/version.rb (hunk 4 above)
Comment (tylersuzukinelson) [inline lib/rails_column_enumerator/version.rb]: It seems to have become Clio codebase standard to add it

File: spec/support/en.yml (hunk 5 above)
Comment (tylersuzukinelson) [inline spec/support/en.yml]: ditto

File: lib/rails_column_enumerator.rb [hunk 6]
```diff
@@ -9,10 +9,10 @@ def self.get_int(val, enum_values, valid_values = nil)
     val
   end
 
-  def self.get_record(val, enum_values, valid_values, localization_values, helper_name)
+  def self.get_record(val, enum_values, valid_values, helper_name, column_name, class_name)
     val = get_int(val, enum_values, valid_values)
     key = valid_values[val]
-    text = localization_values[helper_name].try(:[], key) || localization_values[:default].try(:[], key) || valid_values[val].to_s.titleize
+    text = I18n.t(".models.#{class_name}.#{column_name}.#{helper_name}.#{key}", :default => I18n.t(".models.#{class_name}.#{column_name}.default.#{key}", default: valid_values[val].to_s.titleize))
```
Comment (tylersuzukinelson) [inline lib/rails_column_enumerator.rb]: ```...default: I18n.t...```

File: lib/rails_column_enumerator/version.rb (hunk 4 above)
Comment (brianclio) [inline lib/rails_column_enumerator/version.rb]: ```¯\_(ツ)_/¯```

File: spec/support/en.yml (hunk 5 above)
Comment (brianclio) [inline spec/support/en.yml]: ``` ¯\_(ツ)_/¯ ```

File: README.md (hunk 1 above)
Comment (brianclio) [inline README.md]: I think it's ok as is

**Pull request merged**